gar_exporter.py

```diff
@@ -26,54 +74,52 @@ def _initialize_analyticsreporting(self):
     
     return analytics
 
-  def _get_report(self, analytics):
+  def _get_report(self, analytics,report):
 
     return analytics.reports().batchGet(
-        body={
-          'reportRequests': [
-          {
-            'viewId': VIEW_ID,
-            'dateRanges': [{'startDate': str(os.getenv('START_DATE')), 'endDate': 'today'}],
-          'metrics': [{'expression': 'ga:sessions'}, {'expression': 'ga:pageviews'}, {'expression': 'ga:users'}]
-          }]
-        }
+        body=report
     ).execute()
 
   def _get_metrics(self, response):
     METRIC_PREFIX = 'ga_reporting'
-    LABELS = ['view_id', 'service_email']
-    self._gauges = {}
-
+    LABELS = ['view_id']
+    #self._gauges = {}
+    #print(response)
     for report in response.get('reports', []):
       columnHeader = report.get('columnHeader', {})
       dimensionHeaders = columnHeader.get('dimensions', [])
+      # Added dimentions as labels - fixed bug
+      dimensionHeadersmodified = [x[3:] for x in dimensionHeaders] 
       metricHeaders = columnHeader.get('metricHeader', {}).get('metricHeaderEntries', [])
       rows = report.get('data', {}).get('rows', [])
-
+      testi=0
```

Comment:
Would like to have a better descriptive variable name here. Something like rowNumber or indexRow or whatever makes sense